CircuitPython_Painter/main.py

```diff
@@ -96,9 +96,8 @@ class BMPError(Exception):
 
 except OSError as e:
     if e.args[0] == 28:
-        raise OSError("OS Error 28 0.25")
-    else:
-        raise OSError("OS Error 0.5")
+        raise OSError("OS Error 28 0.25") from e
```

Comment:
@ladyada Just wanted to verify that this is the right change to make. I thought that it was but I'm currently looking at something else where they disabled the check so I wasn't sure which way I should take.

Reply:
Just realized that the top line doesn't match with the bottom. Imagine that they were the same except for the 'from e'

Reply:
also, in some cases where there is not a variable to use, I have been doing the following:

```
- raise OSError("OS Error 0.5")
+ raise OSError("OS Error 0.5") from OSError
```

Reply:
To make pylint happy, you should change `except:` to `except Exception as e:`, so that you always have a `from e` in your raise.  `raise .. from OSError` seems weird to me, though standard Desktop python at least accepts it.

Reply:
Yeah, you're definitely right. I realized this as I was doing it in a few places throughout the libraries but I guess I forgot to do that here.